index.html

```diff
@@ -196,9 +286,30 @@ <h1>About
         <li class='menu'>Skills<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
       </ul>
 
+    </div>
+
+  </section>
+  <section class="contactform">
+    <div class="container"></div>
+    <div class="content-form">
+      <h3>Contact me</h3>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I’d love to help with it </p>
+      <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/xgerlrpz"
+        method="POST">
+        <fieldset id="fs-frm-inputs">
+          <input type="text" name="name" id="full-name" placeholder="Full name" maxlength="30" required>
+          <input type="email" name="_replyto" id="email-address" placeholder="Email address" required>
+          <textarea rows="5" name="message" id="message" placeholder="Enter text here" maxlength="500"
+            required></textarea>
+          <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
+        </fieldset>
+        <input type="submit" value="Get in touch" class="submit-button">
```

Comment:
- [ ] Your button looks good, congrats 👏 . You only need to add one more thing to make it look even better, You have probably forgotten to add button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template: 👇 

![image](https://user-images.githubusercontent.com/55465133/129771641-b76488a8-0ef5-497e-81a5-8028991aff7d.png)

Please let's fix this I am sure you can do this 👍